search/search-domain/src/main/java/io/camunda/search/entities/DecisionInstanceEntity.java

```diff
@@ -11,27 +11,27 @@
 import java.util.List;
 
 public record DecisionInstanceEntity(
-    long key,
-    String id,
+    Long decisionInstanceKey,
```

Comment:
🔧  Since this behaves completely different to all other entities with a key and an ID (key is *not* the PK and not unique), should we document this here?